src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -104,11 +105,49 @@ public async Task<IActionResult> SelectEmployer(SelectEmployerRequest request)
             return View(model);
         }
 
+        [HttpGet]
+        [Route("add/confirm-employer")]
+        [DasAuthorize(ProviderFeature.ProviderCreateCohortV2)]
+        public async Task<IActionResult> ConfirmEmployer(ConfirmEmployerRequest request)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var model = await _modelMapper.Map<ConfirmEmployerViewModel>(request);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("add/confirm-employer")]
```

Comment:
Should this endpoint be called add/create-empty-cohort? The PRG pattern would require it to be confirm-employer then if the model is wrong it would redirect you to the get version of the page. But nothing in the name indicates that it actaully create the empty cohort